wsmaster/codenvy-hosted-license-shared/src/main/java/com/codenvy/license/shared/InvalidLicenseException.java

```diff
@@ -12,7 +12,7 @@
  * is strictly forbidden unless prior written permission is obtained
  * from Codenvy S.A..
  */
-package com.codenvy.api.license;
+package com.codenvy.license.shared;
```

Comment:
Those exceptions has clients in several modules. So it was decided to put it into the shared module.